### DISCUSSION TITLE
Does Anybody Perform Triage or Close Duplicate/Resolved Issues

### DISCUSSION THREAD

**Discussion by sydlexius** [advplyr/audiobookshelf#3190] [General]
I ask because some bugs/requests can take a while to get tagged, while several others get flagged as dupes but haven't been closed.  It ends up making the issue count (as of right now, sitting at 510) appear worse than it really is.  Further, there are almost certainly cases where the really old issues/requests may have been addressed, and an attempt should be made to reach out to the person who filed the issue to confirm if it has been resolved.

**Comment by nichwall** [comment]
We try and stay on top of issues, but it isn't perfect. In the past week there have been 7 issues closed and 9 new issues opened, and 37 issues have been closed in the past 30 days.

If you notice an issue has been resolved/is a duplicate and is still open, a friendly reminder to close the issue is appreciated (either as a GitHub comment or a message in the `dev-chat` channel on Discord). Are there specific open issues you're referring to?

Some issue are kept open even if they aren't necessarily planned to be part of the main project because they end up being duplicate issues and people don't search closed issues before asking for the same thing. The `backlog` label was added a few months ago to help address these issues, but this label hasn't been applied to all issues yet.
